### ISSUE TITLE
New model request: Harmonic DenseNet (ICCV 2019)

### ISSUE THREAD

**Issue by PingoLH** [pytorch/vision#1304]
Hello,
I'd like to introduce our research in efficient CNN model to torchvision.models
Please see our repo:
https://github.com/PingoLH/Pytorch-HarDNet

The new model is 25~35% faster than ResNet running on GPU for both training and inferencing. When being used in object detection such as SSD, Harmonic DenseNet (HarDNet) can further gain accuracy by its enhanced local feature extraction that deploys more layers on high-resolution feature maps. SSD-HarDNet68 achieves a higher COCO mAP than SSD-ResNet101 while the backbone is 27% faster than ResNet-50. It will be our great honor if this work can be accepted by torchvision. Please check out our weight-ready repo above, and let us know if there is any question or concern. Thank you so much!

**Comment by fmassa** [comment]
Hi,

Thanks for opening the issue!

I think it might be better for now to add those models to TorchHub https://pytorch.org/hub .
TorchHub has been built exactly for that, to serve as a place to share newest research models in order to make it easily accessible to everyone.

The models in torchvision are a meant to be references, and we generally wait some time after the paper has been published to have it integrated into torchvision, following a model similar to what PyTorch does for new layers.

Let me know what you think.

**Comment by fmassa** [comment]
HardNet has been added to TorchHub in https://github.com/pytorch/hub/pull/54
I'm closing it for now, let's revisit adding it to torchvision in 6 months
